src/ui/wxWidgets/SafeCombinationPromptDlg.cpp

```diff
@@ -236,20 +273,35 @@ void SafeCombinationPromptDlg::ProcessPhrase()
 
 void SafeCombinationPromptDlg::OnOkClick(wxCommandEvent& WXUNUSED(evt))
 {
-  if (Validate() && TransferDataFromWindow()) {
```

Comment:
Wanted to make the same comment as in `SafeCombinationEntry`, except that here there's no call to `EllipsizeFilePathname()` in the `!TransferDataFromWindow()` case - is that intentional?